code/datums/status_effects/buffs.dm

```diff
@@ -384,3 +384,32 @@
 	if(islist(owner.stun_absorption) && owner.stun_absorption["blooddrunk"])
 		owner.stun_absorption -= "blooddrunk"
 
+/datum/status_effect/sword_spin
+	id = "Bastard Sword Spin"
+	duration = 50
+	tick_interval = 8
+	alert_type = null
+
+
+/datum/status_effect/sword_spin/on_apply()
+	owner.visible_message("<span class='danger'>[owner] begins swinging the sword with inhuman strength!</span>")
+	var/oldcolor = owner.color
+	owner.color = "#ff0000"
+	owner.add_stun_absorption("bloody bastard sword", duration, 2, "doesn't even flinch as the sword's power courses through them!", "You shrug off the stun!", " glowing with a blazing red aura!")
+	owner.status_flags |= GODMODE
```

Comment:
Now, I can think of many rational buffs to add to spinning a eldritch sword around, but I implore you reconsider this particular buff.

Reply:
this has potential for so many funny interactions

Reply:
I'll probably cut it after some tests.

Reply:
:thinking:

Reply:
robustin full of good ideas :joy: